doc/classes/CollisionShape2D.xml

```diff
@@ -18,6 +18,7 @@
 		</member>
 		<member name="one_way_collision" type="bool" setter="set_one_way_collision" getter="is_one_way_collision_enabled" default="false">
 			Sets whether this collision shape should only detect collision on one side (top or bottom).
+			[b]Note:[/b] This property has no effect for [Area2D] nodes.
```

Comment:
I suggest to be more precise and say this property has no effect when this collision shape is used as a child of an `Area2D` node. (Same comment applies to the collision polygon.)